branding/pdf/entities.adoc

```diff
@@ -14,6 +14,7 @@
 :centos: CentOS
 :oracle: Oracle Linux
 :amazon: Amazon Linux
+:alibabaclo: Alibaba Cloud Linux
```

Comment:
clo?

Reply:
I reused the name of the Salt grain, don't blame me for it O:-)

https://github.com/saltstack/salt/blob/master/salt/grains/core.py#L1565